test/suite/test_compact11.py

```diff
@@ -0,0 +1,197 @@
+        #
+        # First convert both bitmaps to a binary string, accounting for any possible leading
+        # zeroes (that would be truncated off). Then compare bit by bit.
+        orig_bits = bin(int('1'+orig, 16))[3:]
+        new_bits = bin(int('1'+new, 16))[3:]
+        self.prout("Original bitmap in binary: " + orig_bits)
+        self.prout("Reopened bitmap in binary: " + new_bits)
+        for o_bit, n_bit in zip(orig_bits, new_bits):
+            if o_bit != '0':
+               self.assertTrue(n_bit != '0')
+
+    def parse_blkmods(self, uri):
+        meta_cursor = self.session.open_cursor('metadata:')
+        config = meta_cursor[uri]
+        meta_cursor.close()
+        # The search string looks like: ,blocks=feffff1f000000000000000000000000
+        # Obtain just the hex string.
+        b = re.search(',blocks=(\w+)', config)
+        self.assertTrue(b)
+        # The bitmap string after the = is in group 1.
+        blocks = b.group(1)
+        self.prout(f"parse_blkmods for '{uri}': {blocks}")
+        return blocks
+
+    def test_compact11(self):
+        # Create N tables.
+        uris = []
+        files = []
+        bitmaps = []
+        for i in range(self.num_tables):
+            uri = self.uri_prefix + f'_{i}'
+            uris.append(uri)
+            files.append(f'file:test_compact11_{i}.wt')
+            self.session.create(uri, self.create_params)
+
+        # Populate the first half of each table.
+        for uri in uris:
+            self.populate(uri, 0, self.table_numkv // 2, self.value_size)
+
+        # Write to disk.
+        self.session.checkpoint()
+
+        os.mkdir(self.backup_incr)
+        self.initial_backup = True
+        self.take_full_backup(self.backup_incr)
+        self.initial_backup = False
+
+        # Insert the latter 50% in each table.
+        for uri in uris:
+            self.populate(uri, self.table_numkv // 2, self.table_numkv, self.value_size)
+
+        # Write to disk.
+        self.session.checkpoint()
+
+        # Delete 50% of each file.
+        for uri in uris:
+            self.delete_range(uri, 50 * self.table_numkv // 100)
+
+        # Write to disk.
+        self.session.checkpoint()
+        for uri in files:
+            bitmaps.append(self.parse_blkmods(uri))
+
+        # Enable background compaction.
+        self.turn_on_bg_compact('free_space_target=1MB')
+
+        # Every time compaction makes progress, perform an incremental backup.
+        bytes_recovered = 0
+        while self.get_files_compacted(uris) < self.num_tables:
+            new_bytes_recovered = self.get_bytes_recovered()
+            if new_bytes_recovered != bytes_recovered:
+                self.prout('Compaction has made progress... ')
+                # Update the incremental backup ID from the parent class.
+                bytes_recovered = new_bytes_recovered
+               
+        assert self.get_files_compacted(uris) == self.num_tables
+        self.turn_off_bg_compact()
```

Comment:
Do we need to turn it off?

Reply:
I have removed it [5138fc6](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/5138fc6cc4875fa9b291fc03b618948222bb21df)